Libraries/PeriphDrivers/Source/TMR/tmr_ai87.c

```diff
@@ -48,10 +48,6 @@ int MXC_TMR_Init(mxc_tmr_regs_t *tmr, mxc_tmr_cfg_t *cfg, bool init_pins)
         break;
 
     case MXC_TMR_IBRO_CLK:
-        if (tmr_id > 3) { // Timers 4-5 do not support this clock source
-            return E_NOT_SUPPORTED;
-        }
-
         clockSource = MXC_TMR_CLK2;
```

Comment:
The IBRO clock source number is different for LPTMR0-1 vs TMR0-3, so you'll still need to add some logic for that:

```
if (tmr_id <= 3) {
    clockSource = MXC_TMR_CLK2;
} else {
    clockSource = MXC_TMR_CLK0;
}
```

![image](https://github.com/user-attachments/assets/a63334c5-0eb2-4f62-a00e-cb00c1140ba1)

Reply:
you are right, fixed